extensions/wikia/Flags/models/Flag.class.php

```diff
@@ -29,8 +29,8 @@ public function getFlagsForPage( $wikiId, $pageId ) {
 
 		if ( !empty( $flagsWithParams ) ) {
 			$flagsParams = $this->getParamsForFlags( $flagsWithParams );
-			foreach ( $flagsParams as $flagId => $flagParams ) {
-				$flagsForPage[ $flagId ]['params'] = $flagParams;
+			foreach ( $flagsParams as $flagTypeId => $flagParams ) {
+				$flagsForPage[ $flagTypeId ]['params'] = $flagParams;
```

Comment:
Perhaps you should raise that idea among engineering and update conventions here https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax